aioredlock/algorithm.py

```diff
@@ -16,35 +16,19 @@ class Aioredlock:
 
     redis_connections = attr.ib(default=[{'host': 'localhost', 'port': 6379}])
 
-    lock_timeout = attr.ib(default=10.0, converter=float)
     # Proportional drift time to the length of the lock
```

Comment:
Drift argument removed but comment still here, move comment to lock() method.

Reply:
cool